src/workerd/io/compatibility-date.capnp

```diff
@@ -8,7 +8,7 @@ using Cxx = import "/capnp/c++.capnp";
 $Cxx.namespace("workerd");
 $Cxx.allowCancellation;
 
-const supportedCompatibilityDate :Text = "2023-01-15";
+const supportedCompatibilityDate :Text = "2023-01-24";
```

Comment:
You don't need to change this in this PR. Changing this will trigger a workerd release.